benchexec/tablegenerator/react-table/src/components/ReactTable.js

```diff
@@ -589,29 +589,20 @@ const Table = (props) => {
       )
       .join(";");
     const value = sort.length ? sort : undefined;
-    const prevParams = getHashSearch();
-    if (prevParams["sort"] !== value) {
-      setHashSearch({ sort: value }, { keepOthers: true });
-    }
+    setURLParameter({ sort: value });
```

Comment:
It is good that we now document the feature of parameter deletion via `undefined` values, because here we can see that it was actually used already!